src/js/app.jsx

```diff
@@ -10,7 +10,11 @@ import {
     faFileAlt,
     faCalendarAlt,
     faFileUpload,
-    faUser
+    faUser,
```

Comment:
I think I missed this in #1182, but I'd like to try to keep this in alphabetical order so it's easy to check if an icon is already imported as the list grows.